include/chemfiles/Connectivity.hpp

```diff
@@ -294,7 +312,7 @@ class Connectivity final {
     const sorted_set<Improper>& impropers() const;
 
     /// Add a bond between the atoms `i` and `j`
-    void add_bond(size_t i, size_t j);
+    void add_bond(size_t i, size_t j, BondOrder bo = UNKNOWN);
```

Comment:
bo => order? Or `bond_order`?

Reply:
Also, could you add the bond order parameter to the documentation?

Reply:
Yes, in may codes `bond_order` is abbreviated as `bo`, but I'm willing to change it!